src/driver/rustc.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright 2012 The Rust Project Developers. See the COPYRIGHT
```

Comment:
I’m wondering what’s the reason for all these wrapper files. Can’t it live in `src/librustc_driver`?

Also the copyright year is quite far in the past.